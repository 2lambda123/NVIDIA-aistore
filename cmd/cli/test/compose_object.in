ais bucket create ais://$BUCKET | awk '{print $1 " " $2}'
echo -n "0123456789" > $OBJECT_1.txt
echo "abcdefghi" > $OBJECT_2.txt

ais object concat $OBJECT_1.txt $OBJECT_2.txt ais://$BUCKET/object.txt

ais object get ais://$BUCKET/object.txt -

mkdir "$OBJECT_3-b"
mkdir "$OBJECT_3-a"
echo -n "2" > $OBJECT_3-b/7.txt
echo -n "0" > $OBJECT_3-b/5.txt
echo -n "1" > $OBJECT_3-b/6.txt
echo -n "4" > $OBJECT_3-a/1.txt
echo -n "3" > $OBJECT_3-a/0.txt
echo "5" > $OBJECT_3.txt

ais object concat $OBJECT_3-b $OBJECT_3-a $OBJECT_3.txt ais://$BUCKET/object1.txt
ais object get ais://$BUCKET/object1.txt -
