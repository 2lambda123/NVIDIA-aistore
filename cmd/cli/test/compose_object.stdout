"$BUCKET" bucket created
COMPOSE "object.txt" [20B] into bucket "$BUCKET"
0123456789abcdefghi
COMPOSE "object1.txt" [7B] into bucket "$BUCKET"
012345